Dockerfile

```
@@ -1,26 +1,24 @@
 # Copyright (c) 2021 LG Electronics Inc.
 # SPDX-License-Identifier: Apache-2.0
-FROM	ubuntu:20.04
+FROM --platform=linux/amd64 python:3.8-slim-buster
```

To provide for **multiple platforms** instead of just one, the setting must be excluded from the `FROM` statement.

```suggestion
FROM python:3.8-slim-buster
```